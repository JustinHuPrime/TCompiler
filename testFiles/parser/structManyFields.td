module foo;

struct bar {
  int x, y;
  float r;
};