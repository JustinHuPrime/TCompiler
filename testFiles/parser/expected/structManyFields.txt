testFiles/parser/input/structManyFields.tc (code):
FILE(1, 1, STAB(ENTRY(bar, STRUCT(testFiles/parser/input/structManyFields.tc, 3, 1, FIELD(int, x), FIELD(int, y), FIELD(float, r)))), MODULE(1, 1, ID(1, 8, foo, REFERENCES())), STRUCTDECL(3, 1, ID(3, 8, bar, REFERENCES(testFiles/parser/input/structManyFields.tc, 3, 1)), VARDECL(4, 3, KEYWORDTYPE(4, 3, int), ID(4, 7, x, REFERENCES()), ID(4, 10, y, REFERENCES())), VARDECL(5, 3, KEYWORDTYPE(5, 3, float), ID(5, 9, r, REFERENCES()))))
